Cargo.lock

```diff
@@ -1421,6 +1434,21 @@ dependencies = [
  "buddy_system_allocator",
 ]
 
+[[package]]
+name = "smoltcp"
```

Comment:
Why are there two smoltcp packages?

Reply:
Sorry, due to the inclusion of `smoltcp` in `ixgbe-driver` for debugging print information, it led to different versions of `smoltcp`. I will replace it with the same version of `smoltcp`.